nubis/bin/example-ldap2consul.conf

```diff
@@ -0,0 +1,13 @@
+[ldap]
+server = ldap.domain.com
+binduser = somebinduser
+bindpass = agoodpass
+searchbase = dc=domain
+searchfilter = (cn=group)
```

Comment:
... this tool is, itself, LDAP specific. The whole point is syncing LDAP to Consul, hence the name, ldap2consul.